### PR TITLE
Adding documentation for Karapace Schema Registry and changed some unsupported versions to the latest versions

### PR DESCRIPTION
**Pull request by YuWang24**
INS-14871

### CODE REVIEW

**Review by harvey-h-yu — APPROVED** [review]
Looks good

File: acc_test/resource_vpc_peering_test.go [hunk 1]
```diff
@@ -2,9 +2,9 @@ package test
 
 import (
 	"fmt"
-//	"io/ioutil"
-//	"os"
-//	"testing"
+	//	"io/ioutil"
```
Comment (WeiXinFam-Insta) [inline acc_test/resource_vpc_peering_test.go:5]: **[QUESTION]** Was this spacing intended?

File: examples/main.tf [hunk 2]
```diff
@@ -189,6 +189,40 @@ resource "instaclustr_cluster" "example_kafka" {
   wait_for_state = "RUNNING" // the required state of the cluster before doing the bundle user password updates
 }
 
+// Use Kafka with Karapace Schema Registry
+resource "instaclustr_cluster" "example_kafka_with_karapace_schema_registry" {
+  cluster_name = "test_kafka_with_karapace"
+  node_size = "KFK-PRD-r6g.large-250"
+  data_centre = "US_WEST_2"
+  sla_tier = "NON_PRODUCTION"
```
Comment (WeiXinFam-Insta) [inline examples/main.tf:197]: **[QUESTION]** Isn't `KFK-PRD-r6g.large-250` production node size?

**Review by WeiXinFam-Insta — COMMENTED** [review]
Left some questions

File: acc_test/resource_vpc_peering_test.go (hunk 1 above)
Comment (YuWang24) [reply acc_test/resource_vpc_peering_test.go:5]: It appears after merging master branch, but in my local I don't have these changes. Should be fine.

File: examples/main.tf (hunk 2 above)
Comment (YuWang24) [reply examples/main.tf:197]: Yeah, it's just an example, I don't think it does matter. We also have production node sizes in other examples.

Comment (WeiXinFam-Insta) [reply examples/main.tf:197]: i prefer consistency but okays.

**Review by WeiXinFam-Insta — APPROVED** [review]
LGTM

**Review by harvey-h-yu — APPROVED** [review]

**Review by harvey-h-yu — DISMISSED** [review]

**Review by arjunrajinstaclustr — DISMISSED** [review]

**Review by arjunrajinstaclustr — APPROVED** [review]

**Review by WeiXinFam-Insta — APPROVED** [review]
lgtm

**Review by harvey-h-yu — APPROVED** [review]

**Pull request merged**